lib/net/hostlist.js

```diff
@@ -552,7 +592,7 @@ class HostList {
       for (let i = 0; i < entry.refCount; i++)
         factor *= 2;
 
-      if (random(factor) !== 0)
+      if (this.random(factor) !== 0)
```

Comment:
suggestion: add `{}` around this clause because the `else` on the next line confused me for a sec, even though the indentation is a clue